src/spline2/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+# add apps XYZ.cpp, e.g., qmc_particles.cpp
```

Comment:
Things in spline are helper functions for using einspline in EinsplineSet. Spline2 is the new 3D spline implementation to replace things in einspline folder. When we grave einspline, spline should also go away but not merged with spline2.